src/components/AppMapDetailsObj.ts

```diff
@@ -415,4 +418,100 @@ export default class AppMapDetailsObj extends AppMapDetailsBase<MapMarkerObj | M
     }
     return "Normal";
   }
+  korokType() {
```

Comment:
isKorok? `korokType` makes it sound like it's returning a Korok type, not whether the object is a Korok.